### PR TITLE
Create Sopes.md

### PR DESCRIPTION
**Pull request by jprmz**

### CODE REVIEW

File: mains/Sopes.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Ingredients:
+
+## For the sopes:
+2 cups masa harina (corn flour, like Maseca) SDSAJKDJSAKHLFC
```
Comment (Gajera0703) [inline mains/Sopes.md:4]: please remove this bunch of letters.
```suggestion
2 cups masa harina (corn flour, like Maseca)
```

File: mains/Sopes.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+1 1/4 cups warm water (adjust as needed) DFLASLDFKJASÖD
```
Comment (Gajera0703) [inline mains/Sopes.md:5]: there are some extra letters.
```suggestion
1 1/4 cups warm water (adjust as needed)
```

**Review by Gajera0703 — CHANGES_REQUESTED** [review]
there are few

**Review by Camilogiu — COMMENTED** [review]
I love it!

**Pull request merged**